.gitignore

```diff
@@ -1,10 +1,3 @@
-## Compiled JavaScript files
-**/*.js
```

Comment:
Why is this being removed?

Reply:
Too generic. It was causing top-level .js files to be ignored. The lib folder is already ignored